lib/stripe/dispute.rb

```diff
@@ -10,7 +10,7 @@ class Dispute < APIResource
     custom_method :close, http_verb: :post
 
     def close(params = {}, opts = {})
-      resp, opts = request(:post, close_url, params, opts)
+      resp, opts = request(:post, resource_url + "/close", params, opts)
```

Comment:
Can you add a comment to mark the `close_url` method as deprecated, since it's now useless?